backend/app/views/spree/admin/payments/_list.html.erb

```diff
@@ -56,12 +56,16 @@
               <%= link_to_with_icon 'edit', t('spree.actions.edit'), nil, no_text: true, class: "js-edit", data: {action: 'edit'} %>
             <% end %>
             <% allowed_actions = payment.actions.select { |a| can?(a.to_sym, payment) } %>
+            <%= allowed_actions %>
             <% allowed_actions.each do |action| %>
-              <% if action == 'credit' %>
+              <% case action %>
```

Comment:
I prefer `case` to `if` clauses when there are multiple `when`, but maybe in this context (only one `when`) `if` would be a nicer choice? 

My preference would go to `if` because while they both work fine here, using `case` produces 2 lines of ERB (maybe it's just me, but the less ERB the better 😄) when one suffices with `if`.